pkg/inspecttypes/dockercompat/dockercompat.go

```diff
@@ -106,7 +106,7 @@ type Container struct {
 	// TODO: SizeRw     *int64 `json:",omitempty"`
 	// TODO: SizeRootFs *int64 `json:",omitempty"`
 
-	// TODO: Mounts          []MountPoint
+	Mounts []specs.Mount
```

Comment:
This should be `[]MountPont`, not `[]specs.Mount` for Docker compatibility

https://github.com/moby/moby/blob/f0014860c1b3345e1fcc7ed81c491298de2633fb/api/types/types.go#L416-L427
https://github.com/opencontainers/runtime-spec/blob/6969a0a09ab162a574cafcc9ac814e498962c943/specs-go/config.go#L110-L120

Reply:
Addressed in https://github.com/containerd/nerdctl/pull/822